### PR TITLE
add "detach elements" rule

### PR DESCRIPTION
**Pull request by Ex-iT**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,8 +16,25 @@ This guide aims to improve the way your team uses [jQuery](http://jquery.com/).
...
-
+* [Detach elements when working with them](#detach-elements-when-working-with-them)
 
 ## About jQuery
 
 [jQuery](http://jquery.com/) is a utility library for easy DOM access & manipulation, event handling, Ajax and more. By using jQuery you can write consise and expressive code which works across modern and legacy browsers. jQuery has extensive tests, detailed documentation, a large active community and an ecosystem of plugins.
+
+## Detach elements when working with them
+### Why?
+Because updating the DOM is heavy on the client elements should be .detach() -ed when updating multiple things.
```
Comment (jbmoelker) [inline README.md]: link to jQuery docs

File: README.md [hunk 2]
```diff
@@ -16,8 +16,25 @@ This guide aims to improve the way your team uses [jQuery](http://jquery.com/).
...
+
+### How?
+``` javascript
+var todoItems = $('.todo-items');
```
Comment (jbmoelker) [inline README.md]: `$todoItems` or simply `$items`

File: README.md [hunk 3]
```diff
@@ -16,8 +16,25 @@ This guide aims to improve the way your team uses [jQuery](http://jquery.com/).
 ## Table of Contents
 
 * [About jQuery](#about-jquery)
-
+* [Detach elements when working with them](#detach-elements-when-working-with-them)
```
Comment (jbmoelker) [inline README.md]: "Use `.detach()` before heavy DOM operations"?

**Comment by jbmoelker** [comment]
I've personally never used this. Have you? Is it effective? Experience any issues? Like rearrangements of elements after appending again?

**Comment by jbmoelker** [comment]
@jpsc I'm still struggling with this one. Has anyone ever had performance issues with jQuery? Did this solve it? Is there an article or something somewhere we can refer to?

**Comment by jpsc** [comment]
The only thing I could find was:

> The .detach() method is useful when removed elements are to be reinserted into the DOM at a later time because it keeps all data and events.

It does remove the element from DOM like .remove() but stores the meta information.

**Comment by jpsc** [comment]
https://learn.jquery.com/performance/detach-elements-before-work-with-them/
Its like a document fragment.
But since was never experienced performance issues when not using it me and @Ex-iT decided to leave it out
